python/GafferImageUI/ImageViewUI.py

```diff
@@ -1167,6 +1180,43 @@ def _updateFromPlug( self ) :
 		for i in self.__row[1:]:
 			i.setEnabled( m != "" )
 
+	def __dragEnter( self, tabbedContainer, event ) :
+
+		result = False
+		if isinstance( event.data, Gaffer.Node ) :
+			result = True
+		elif isinstance( event.data, Gaffer.Set ) :
+			if event.data.size() and isinstance( event.data[0], Gaffer.Node ) :
+				result = True
+
+		if result :
+			for i in self.__row:
+				i.setHighlighted( True )
+
+		return result
```

Comment:
I think we probably want to `return True` here, so we never have to worry about weird interactions with the drag handling from PlugValueWidget.

Reply:
I'm not aware of other places in Gaffer where we make a drag event return True in the case where we don't actually want to handle the drop, but we want to prevent the default handler from accepting it.

I've done this by returning True on invalid drags, but not setting highlighted.  Does this seem right?

Reply:
Another note here:  I'm now treating drags of nodes with no valid image plugs the same as drags of things that aren't nodes:  I return True to suppress further processing, but don't highlight or accept the drop.  Does that seem right?

Reply:
That seems reasonable to me